examples/hcp-eks-demo/main.tf

```diff
@@ -8,7 +8,7 @@ module "vpc" {
   cidr                 = "10.0.0.0/16"
   azs                  = data.aws_availability_zones.available.names
   private_subnets      = ["10.0.1.0/24", "10.0.2.0/24", "10.0.3.0/24"]
-  public_subnets       = []
+  public_subnets       = ["10.0.4.0/24", "10.0.4.0/24", "10.0.5.1/24"]
```

Comment:
Turns out even though the example doesn't use the public subnets, the aws vpc module needs it.

Reply:
And now we do use it.